pvacseq/lib/coverage_filter.py

```diff
@@ -4,40 +4,62 @@
-    parser.add_argument('input_file', type=argparse.FileType('r'),
-                        help="Output file from binding filter with read count info appended")
-    parser.add_argument('output_file', type=argparse.FileType('w'),
-                        help="Filtered file")
-    parser.add_argument('--normal-cov', type=int,
-                        help="Normal Coverage Cutoff. Sites above this cutoff will be considered. " +
-                        "default 5",
-                        default=5)
-    parser.add_argument('--tdna-cov', type=int,
-                        help="Tumor DNA Coverage Cutoff. Sites above this cutoff will be considered. " +
-                        "default 10",
-                        default=10)
-    parser.add_argument('--trna-cov', type=int,
-                        help="Tumor RNA Coverage Cutoff. Sites above this cutoff will be considered. " +
-                        "default 10",
-                        default=10)
-    parser.add_argument('--normal-vaf', type=int,
-                        help="Normal VAF Cutoff. Sites BELOW this cutoff in normal will be considered. " +
-                        "default 2",
-                        default=2)
-    parser.add_argument('--tdna-vaf', type=int,
-                        help="Tumor DNA VAF Cutoff. Sites above this cutoff will be considered. " +
-                        "default 40",
-                        default=40)
-    parser.add_argument('--trna-vaf', type=int,
-                        help="Tumor RNA VAF Cutoff. Sites above this cutoff will be considered. " +
-                        "default 40",
-                        default=40)
-    parser.add_argument('--expn-val', type=int,
-                        help="Gene Expression (FPKM) Cutoff. " +
-                        "default 1",
-                        default=1)
+    parser.add_argument(
+        'input_file', type=argparse.FileType('r'),
+        help="The final report .tsv file to filter"
+    )
+    parser.add_argument(
+        'output_file', type=argparse.FileType('w'),
+        help="Output .tsv file containing list of filtered epitopes based on coverage and expression values"
+    )
+    parser.add_argument(
+        '--normal-cov', type=int,
+        help="Normal Coverage Cutoff. Sites above this cutoff will be considered. "
+             + "Default: 5",
+        default=5
+    )
+    parser.add_argument(
+        '--tdna-cov', type=int,
+        help="Tumor DNA Coverage Cutoff. Sites above this cutoff will be considered. "
+             +"Default: 10",
+        default=10
+    )
+    parser.add_argument(
+        '--trna-cov', type=int,
+        help="Tumor RNA Coverage Cutoff. Sites above this cutoff will be considered. "
+             + "Default: 10",
+        default=10
+    )
+    parser.add_argument(
+        '--normal-vaf', type=int,
+        help="Normal VAF Cutoff. Sites BELOW this cutoff in normal will be considered. "
+             + "Default: 2",
```

Comment:
I think it's to emphasize it since it's different from the other vaf cuttoff behavior.